doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -855,7 +855,10 @@ A subscription is represented by a JSON object with the following fields:
 + `id`: Subscription unique identifier. Automatically created at creation time.
 + `subject`: It is an object that describes the subject of the subscription.
 + `notification`: It is an object that describes the notification received by the subscriber.
-+ `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
++ `expires`: Subscription expiration date in ISO8601 format. Permanent subscriptions must omit
+   this field.
```

Comment:
When the referred standard describes the behavior I think the best approach is not including the details in the document pointing to the standard for two main reasons:

* Avoid potential contractions (noboby will provide a better description than the standard itself)
* Avoid making the document unnecesarry long or complex

In this particular case, I think is pretty clear: use ISO8601 dates. On the one hand, within the ISO8601 specification I think is pretty clear what a date is and what a interval (duration) is. On the other hand, think we are very "biassed" to the "P1M" notation because we have suffered a long time :) but most of poeple doesn't think in duration when the think ISO8601. Thus, I don't see the need to stress this.

However, it is not so important in this PR to have a perfect wording for this, as I said in a previous comment. We don't know yet all the details on how the ISO8601 stuff will be implemented at the end.